bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/IgrinsBlueprint.scala

```diff
@@ -0,0 +1,31 @@
+package edu.gemini.model.p1.immutable
+
+import edu.gemini.model.p1.mutable.IgrinsBlueprintChoice
+import edu.gemini.model.p1.{mutable => M}
+
+object IgrinsBlueprint {
+  def apply(m: M.IgrinsBlueprint): IgrinsBlueprint = new IgrinsBlueprint(m)
+}
+
+case class IgrinsBlueprint() extends GeminiBlueprintBase {
+  override val name: String = "IGRINS"
+  override val visitor = true
```

Comment:
`: Boolean`

(I know, it's obvious here, but I think it's a good idea to document the expected type of all public methods.)